src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,131 @@
+.watch {
+  display: flex;
+
+  height: 400px;
```

Comment:
A lot of hardcoded values. Think about how to decrease the such amount. F.e. calculate them, use flexbox props, use relative units etc.